server/db/models/User.model.ts

```diff
@@ -0,0 +1,156 @@
+import db from "../db.js";
+import bcrypt from "bcrypt";
+import jwt from "jsonwebtoken";
+import { Model, DataType } from "sequelize-typescript";
+
+declare class UserModel extends Model {
```

Comment:
This is written as a class; however, I think an interface would be more appropriate.